MyPy Errors: "Skipping analyzing "py._path.local": module is installed, but missing library stubs or py.typed marker"

> type_check run-test: commands[1] | mypy --show-error-codes tests
tests/test_geschaeftspartner.py:4: error: Skipping analyzing "py._path.local": module is installed, but missing library stubs or py.typed marker  [import]
tests/test_adresse.py:4: error: Skipping analyzing "py._path.local": module is installed, but missing library stubs or py.typed marker  [import]
tests/test_adresse.py:4: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports

Source: https://github.com/bo4e/BO4E-python/pull/425 but also occurs on latest main (v3.10):
https://github.com/bo4e/BO4E-python/actions/runs/3377394980/jobs/5606268003

--- a/docs/uml.py
+++ b/docs/uml.py
@@ -41,7 +41,7 @@ class _Package:
     scope: List[str] | None
     color: str | None
 
-    def __init__(self, *, scope: List[str] = None, color: str = None):
+    def __init__(self, *, scope: Optional[List[str]] = None, color: Optional[str] = None):
         self.scope = scope
         self.color = color
 

--- a/docs/uml.py
+++ b/docs/uml.py
@@ -119,7 +119,7 @@ class _UMLNetworkABC(nx.MultiDiGraph, metaclass=ABCMeta):
         super().add_node(
             node,
             cls=cls,
-            fields=cls.__fields__.copy() if hasattr(cls, "__fields__") else {},  # type:ignore[attr-defined]
+            fields=cls.__fields__.copy() if hasattr(cls, "__fields__") else {},
         )
 
     def add_extension(self, node1: str, node2: str) -> None:

--- a/tests/serialization_helper.py
+++ b/tests/serialization_helper.py
@@ -1,14 +1,14 @@
 """
 A module that simplifies assertions for json serialization
 """
-from typing import Any, Dict, TypeVar
+from typing import Any, Dict, Optional, TypeVar
 
 from dictdiffer import diff  # type: ignore[import]
 
 T = TypeVar("T")
 
 
-def assert_serialization_roundtrip(serializable_object: T, expected_json_dict: Dict[str, Any] = None) -> T:
+def assert_serialization_roundtrip(serializable_object: T, expected_json_dict: Optional[Dict[str, Any]] = None) -> T:
     """
     Serializes the serializable_object using the provided schema,
     then asserts, that the result is equal to the expected_json_dict (if given),

--- a/tests/test_adresse.py
+++ b/tests/test_adresse.py
@@ -1,7 +1,7 @@
 from typing import Dict, Optional
 
 import pytest
-from py._path.local import LocalPath
+from py._path.local import LocalPath  # type:ignore[import]
 from pydantic import ValidationError
 
 from bo4e.com.adresse import Adresse

--- a/tests/test_geschaeftspartner.py
+++ b/tests/test_geschaeftspartner.py
@@ -1,7 +1,7 @@
 import json
 
 import pytest
-from py._path.local import LocalPath
+from py._path.local import LocalPath  # type:ignore[import]
 from pydantic import ValidationError
 
 from bo4e.bo.geschaeftspartner import Geschaeftspartner

